### ISSUE TITLE
Homebrew fails to install storyscript

### ISSUE THREAD

**Issue by eddiemoore** [storyscript/homebrew-brew#14]
Ran `brew install storyscript/brew/story`

Getting this when trying to install:
```shell
Last 15 lines from /Users/edmoore/Library/Logs/Homebrew/story/01.python3:
-c
import setuptools, tokenize
__file__ = 'setup.py'
exec(compile(getattr(tokenize, 'open', open)(__file__).read()
  .replace('\r\n', '\n'), __file__, 'exec'))
--no-user-cfg
install
--prefix=/private/tmp/story--homebrew-virtualenv-20190920-71264-tzcdmq/target
--install-scripts=/private/tmp/story--homebrew-virtualenv-20190920-71264-tzcdmq/target/bin
--single-version-externally-managed
--record=installed.txt

Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'setuptools'
```

**Comment by judepereira** [comment]
hey @eddiemoore, which version of homebrew are you using?

**Comment by eddiemoore** [comment]
@judepereira im using Homebrew 2.1.11

**Comment by wilzbach** [comment]
Seems to be a common issue, e.g.
- https://github.com/datawire/homebrew-blackbird/issues/7#issuecomment-416746467
- https://github.com/Homebrew/homebrew-python/issues/365#issuecomment-260109604
- https://github.com/Homebrew/homebrew-python/issues/216#issuecomment-91119916
- https://github.com/jbarlow83/homebrew-ocrmypdf/issues/6#issuecomment-358820853

tl;dr: looks like a problem with @eddiemoore's Python installation. @eddiemoore maybe you could try following the advice in the first issue?
